Wikipedia/Code/ArticleRightAlignedImageCollectionViewCell.swift

```diff
@@ -50,6 +54,11 @@ open class ArticleRightAlignedImageCollectionViewCell: ArticleCollectionViewCell
             origin.y += saveButtonFrame.height
         }
         origin.y += margins.bottom
+        
+        if (apply) {
+            separator.frame = CGRect(x: 0, y: origin.y - 0.5, width: size.width, height: 0.5)
```

Comment:
Can we use `1.0 / UIScreen.main.scale` here so non-retina devices will get single pixel hairlines too?